react-datetime.d.ts

```diff
@@ -50,17 +50,17 @@ declare module ReactDatetime {
      only parameter, if the date in the input is valid. If it isn't, the value
      of the input (a string) is returned.
      */
-    onChange?:(x: string) => void;
+    onChange?:(x: string|any) => void;
```

Comment:
Please add space between `onChange?:` and `(x: string ...)` to comply with styling of the rest of the file.